### PR TITLE
Login format sh

### PR DESCRIPTION
**Pull request by seolheekang1221**
# Link to Trello
- Link goes here

# Proposed Changes
- change 1
- change 2

# Additional Info
- any additional information or context

# Checklist
- [o] Comments in the code
- [] Remove console.logs
- [] Clean code

### CODE REVIEW

**Review by sal59265 — APPROVED** [review]
There is a conflict, please resolve before merging.

File: client/src/App.js [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
+  const [error, setError] = useState("");
+
+  const Login = (forms) => {
+    console.log(forms);
+
+    if (
+      forms.email == loginUser.email &&
+      forms.password == loginUser.password
+    ) {
+      console.log("Success");
```
Comment (jingu-k816) [inline client/src/App.js:20]: please remove console.log

File: client/src/App.js [hunk 2]
```diff
@@ -1,25 +1,51 @@
...
+      setEmail({
+        email: forms.email,
+        password: forms.password,
+      });
+    } else {
+      console.log("your password is incorret");
```
Comment (jingu-k816) [inline client/src/App.js:26]: don't need console log here as well

File: client/src/App.js [hunk 3]
```diff
@@ -1,25 +1,51 @@
...
+  };
+
+  const [email, setEmail] = useState({ email: "", password: "" });
+  const [error, setError] = useState("");
+
+  const Login = (forms) => {
+    console.log(forms);
+
+    if (
+      forms.email == loginUser.email &&
```
Comment (jingu-k816) [inline client/src/App.js:17]: type checking should be triple equals not double

File: client/src/App.js [hunk 4]
```diff
@@ -1,25 +1,51 @@
...
+      forms.password == loginUser.password
```
Comment (jingu-k816) [inline client/src/App.js:18]: same comment as above

File: client/src/App.js [hunk 5]
```diff
@@ -1,25 +1,51 @@
...
+  const loginUser = {
+    email: "seolhee@gmail.com",
+    password: "seolhee",
+  };
+
+  const [email, setEmail] = useState({ email: "", password: "" });
+  const [error, setError] = useState("");
+
+  const Login = (forms) => {
+    console.log(forms);
```
Comment (jingu-k816) [inline client/src/App.js:14]: no console.logs

**Review by jingu-k816 — CHANGES_REQUESTED** [review]
We need to re do the login page from scratch

**Pull request closed without merging**